.travis.yml

```diff
@@ -33,4 +34,4 @@ script:
   - mkdir build && cd build
   - cmake ../
   - cmake --build .
-  - ./runUnitTests
\ No newline at end of file
+  - valgrind ./runUnitTests
```

Comment:
we should consider running our test always with valgrind but because of glfw there is a lot of garbage in output so maybe this should be done only when necessary.